sci-misc/boinc/boinc-7.2.0-r1.ebuild

```diff
@@ -23,7 +23,7 @@ RDEPEND="
 	!sci-misc/boinc-bin
 	!app-admin/quickswitch
 	>=app-misc/ca-certificates-20080809
-	dev-libs/openssl
+	dev-libs/openssl:=
```

Comment:
Actually this should be "dev-libs/openssl:0". The other slot only exists for binary compatibility and should not be used for normal packages.